src/en/charms-storage.md

```diff
@@ -164,60 +172,72 @@ For example, as described above, attaching an EBS volume to an EC2 instance
 requires that they both reside within the same availability zone. If this is
 not the case, Juju will return an error.
 
-#### Adding and detaching storage
-
-Assuming the storage provider supports it, storage can be created and
-dynamically attached to a unit using `juju add-storage`.
+### Adding and detaching storage
 
-For example, to create a 100GiB EBS volume and attach it to unit 'postgresql/0'
-as its pgdata storage:
-
-```bash
-juju add-storage postgresql/0 pgdata=ebs,100G
-```
+Assuming the storage provider supports it, storage can be created and attached
+to a unit using `juju add-storage`. Juju will ensure the storage is allowed to
+attach to the unit's machine.
 
-Juju will ensure the storage is allowed to attach to the unit's machine. In the
-above example, the EBS volume was created in the same availability zone as the
-instance (a requirement).
+!!! Note:
+    Currently, it is not possible to add storage to the model without also
+    attaching it to a unit.
 
 Charms can specify a maximum number of storage instances. In the case of the
-postgresql charm, a maximum of one is allowed for 'pgdata'. If an attempt is
+charm 'postgresql', a maximum of one is allowed for 'pgdata'. If an attempt is
 made to exceed it, Juju will return an error.
 
 Dynamic storage can be detached from units using `juju detach-storage`.
 
-For example, to detach storage 'pgdata/0' from unit 'postgresql/0':
-
-```bash
-juju detach-storage postgresql/0 pgdata/0
-```
-
 Charms can also define a minimum number of storage instances. The postgresql
 charm specifies a minimum of zero for 'pgdata' whereas another charm may specify
 a different number. In any case, if detaching storage from a unit would bring
 the total number of storage instances below the minimum, Juju will return an
 error.
 
-#### Persistence
+#### Examples
+
+To create a 32GiB EBS volume and attach it to unit 'ceph-osd/0' as its OSD
+storage:
+
+```bash
+juju add-storage ceph-osd/0 osd-devices=ebs,32G,1
+```
+
+Above, the volume was created in the same availability zone as the instance (a
```

Comment:
Right. It's because it's already been said two times by this time. Here we're just stating what happened in this example. I was actually going to remove this sentence altogether.